src/main/java/se/sundsvall/billingdatacollector/integration/opene/mapper/BillingRecordConstants.java

```diff
@@ -6,6 +6,7 @@ private BillingRecordConstants() {
 		throw new IllegalStateException("Utility class");
 	}
 
+	public static final String SUNDSVALLS_MUNICIPALIY_ID = "2281";
```

Comment:
This is misspelled. Same with row 10.

Also fix spelling on row 11.

Reply:
Fixed